integration/render_test.go

```diff
@@ -76,7 +77,7 @@ spec:
 		t.NewTempDir().
 			Write("deployment.yaml", test.input).
 			Chdir()
-		deployer, err := deploy.NewKubectlDeployer(&runcontext.RunContext{
+		deployer, err := kubectl.NewKubectlDeployer(&runcontext.RunContext{
```

Comment:
nit: Rename `NewKubectlDeployer` to `NewDeployer` or `New` according to the golang package naming convention https://golang.org/doc/effective_go.html#package-names.

Reply:
yeah meant to do this. updating for all deployers